docs/design-system/ui_kits/ovp/graph.html

```
@@ -0,0 +1,470 @@
+</style>
+</head>
+
+<body>
+<div class="graph-page" id="graph-page">
+
+  <div id="graph-canvas"></div>
+
+  <!-- ============ TOP BAR ============ -->
+  <header class="topbar">
+    <span class="brand">
+      <img src="../../assets/monogram.svg" width="20" height="20" style="display:block">
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)